clarifai/models/model_serving/docs/inference_parameters.md

```diff
@@ -0,0 +1,134 @@
+## Inference paramaters
+
+When making prediction, you may need to change some paramaters to adjust the result. Those paramaters will be passed through `paramaters()` of a request in triton python model.
+
+In order to send it to `**kwargs` of `get_predictions` in `inference.py`, you can define some parameters and they will be visible and adjustable on Clarifai model view.
+
+This document helps you to create your inference parameters that can be visibale and adjustable easily on Clarifai platform. The defined parameters will be sent as `json` file when you use `clarifai-upload-model` cli.
+
+### JSON file structure:
+The file contains a list of object has 4 fields:
+* `path` (str): name of your parameter, it must be valid as python variable
+* `field_type` (int): the parameter data type is one of {1,2,21,3}, it means {boolean, string, encrypted_string, number} respectively. `Number` means `int` or `float`. `Encrypted_string` is simple a string can be used to store your secrets like API key...
```

Comment:
"Encrypted_string is simple a string can be used to store your secrets like API key..."
->
"Encrypted_string is a string that can be used to store your secrets, like API key. The API will not return the values for this as plaintext."

Reply:
thanks for your suggestion